src/libraries/Microsoft.PowerFx.Core/Public/Values/CustomObjectValue.cs

```diff
@@ -23,6 +23,8 @@ public interface ICustomObject
 
         bool IsNumber { get; }
 
+        bool IsBoolean { get; }
```

Comment:
are all these flags exclusive? What do you think of making this a single enum?

Reply:
Changed to an enum